src/modules/UI/scenes/scenesReducer.test.js

```diff
@@ -42,6 +42,11 @@ test('initialState', () => {
         publicAddress: ''
       }
     },
+    requestType: {
+      useLegacyAddress: false,
+      receiveAddress: {},
```

Comment:
do not store duplicated data in redux
this data is stored elsewhere

Reply:
This is in the scene reducer test and is consistent with where the redux data is stored and how the test works. Please explain what is wrong . This was necessary to pass tests.

Reply:
ok, I see what you mean, receive address is not needed actually.. it is removed now